sql-cli/tests/test_project.py

```diff
@@ -1,29 +1,22 @@
-import tempfile
+from tempfile import TemporaryDirectory
 
 from sql_cli.project import BASE_SOURCE_DIR, Project
 from tests.utils import list_dir
 
-BASE_PATHS_COUNT = 19
 BASE_PATHS = [obj for obj in list_dir(str(BASE_SOURCE_DIR)) if not obj.stem == ".gitkeep"]
 
 
 def test_initialise_project_with_dirname():
-    with tempfile.TemporaryDirectory() as dir_name:
-        assert not list_dir(dir_name)
+    with TemporaryDirectory() as dir_name:
         Project(dir_name).initialise()
-        new_dir_files_list = list_dir(dir_name)
-        assert new_dir_files_list == BASE_PATHS
-        assert len(new_dir_files_list) == BASE_PATHS_COUNT
```

Comment:
I would rather we continue checking the number of files as well - since there is a risk someone may update the `BASE_SOURCE_DIR` to an empty directory